src/DynamicObj/DynamicObj.fs

```diff
@@ -39,12 +36,43 @@ type DynamicObj() =
             | :? 'a -> o :?> 'a |> Some
             | _ -> None
 
+
+    member this.TryGetStaticPropertyInfo name : PropertyHelper option  = 
+        ReflectionUtils.tryGetStaticPropertyInfo this name
+
+    member this.TryGetDynamicPropertyInfo name : PropertyHelper option =
+        #if FABLE_COMPILER_JAVASCRIPT  || FABLE_COMPILER_TYPESCRIPT
+        FableJS.tryGetDynamicPropertyHelper this name
+        #endif
+        #if FABLE_COMPILER_PYTHON
+        FablePy.tryGetDynamicPropertyHelper this name
+        #endif
+        #if !FABLE_COMPILER
+        match properties.TryGetValue name with            
+        | true,_ -> 
+            Some {
+                Name = name
+                IsStatic = false
+                IsDynamic = true
+                IsMutable = true
+                IsImmutable = false
+                GetValue = fun o -> properties.[name]
+                SetValue = fun o v -> properties.[name] <- v
+                RemoveValue = fun o -> properties.Remove(name) |> ignore
+            }
+        | _      -> None
+        #endif
         
+    member this.TryGetPropertyInfo name : PropertyHelper option =
+        match this.TryGetStaticPropertyInfo name with
+        | Some pi -> Some pi
+        | None -> this.TryGetDynamicPropertyInfo name
+
     /// Sets property value, creating a new property if none exists
     member this.SetValue (name,value) = // private
         // first check to see if there's a native property to set
         
-        match ReflectionUtils.tryGetPropertyInfo this name  with
+        match ReflectionUtils.tryGetStaticPropertyInfo this name  with
```

Comment:
Now that you have `this.TryGetStaticPropertyInfo`, can you use it here instead of an external module?

Reply:
What module do you mean?

Reply:
ReflectionUtils.tryGetStaticPropertyInfo is used although a method with the exact same name exists on the class. On other memberst, you used the instance method instead of the functions from `Reflectionutils`

Reply:
For example, you used `this.TryGetPropertyInfo` in the Remove method instead of `ReflectionUtils.tryGetPropertyInfo`